app/src/main/java/de/westnordost/streetcomplete/data/osm/edits/update_tags/StringMapChangesBuilder.kt

```diff
@@ -24,15 +24,15 @@ class StringMapChangesBuilder(private val source: Map<String, String>) {
     }
 
     fun add(key: String, value: String) {
-        require(!source.containsKey(key)) { "The key '$key' already exists in the map." }
+        require(!source.containsKey(key)) { "The key '$key' already exists in the map '$source." }
```

Comment:
Add a missing quote:
```suggestion
        require(!source.containsKey(key)) { "The key '$key' already exists in the map '$source'." }
```